### PR TITLE
Add README.md for kinesis exporter.

### PR DESCRIPTION
**Pull request by dengliming**
**Description:** Add README.md for kinesis exporter.
for [https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/317](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/317)

### CODE REVIEW

File: exporter/kinesisexporter/README.md [hunk 1]
```diff
@@ -1,3 +1,64 @@
 # Kinesis Exporter
 
-To be added.
\ No newline at end of file
+This exporter can be used to send metrics to Kinesis.
```
Comment (anuraaga) [inline exporter/kinesisexporter/README.md:3]: Looked at the code and it seems to be traces not metrics

File: exporter/kinesisexporter/README.md [hunk 2]
```diff
@@ -1,3 +1,64 @@
...
+
+The following configuration options are supported:
+
+- `queue_size`: Dictates how many spans the queue should support. default: `100000`
```
Comment (anuraaga) [inline exporter/kinesisexporter/README.md:7]: Which queue / workers are these? I guess the collector has queuing logic so it's hard to follow what these parameters actually control.

**Review by anuraaga — COMMENTED** [review]
Thanks!

File: exporter/kinesisexporter/README.md (hunk 1 above)
Comment (dengliming) [reply exporter/kinesisexporter/README.md:3]: yes. you were right.

File: exporter/kinesisexporter/README.md (hunk 2 above)
Comment (dengliming) [reply exporter/kinesisexporter/README.md:7]: I'm also a little confused here. It seems should used by exporter. but I can't see any test about it. I would really appreciate it If you can help to take a deep see on it. : )

**Comment by tigrannajaryan** [comment]
@anuraaga it appears this is waiting for your response to a comment.

**Comment by tigrannajaryan** [comment]
@anuraaga @dengliming please make progress on the PR or close it.
